libraries/gpu-gl-common/src/gpu/gl/GLBackend.cpp

```diff
@@ -152,6 +209,24 @@ void GLBackend::init() {
     });
 }
 
+size_t GLBackend::getAvailableMemory() {
```

Comment:
seems like there's some repetition between this code and the above.  could we DRY it up by getting the video card first, and then calling this method?

Reply:
What do you mean? It performs a different function (available vs total)